config/plugins/visualizations/kepler/config/kepler.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="Kepler.gl" embeddable="true">
+    <description>Geospatial analysis tool for large-scale data sets</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test test_attr="ext" result_type="datatype">csv</test>
```

Comment:
I would not open any CSV file with this Vis. Either we add this geocsv as a separate datatype or we exclude it?

Reply:
Ok, I will follow-up with adding `geocsv`.